packages/xdl/src/detach/AndroidShellApp.js

```diff
@@ -132,9 +132,9 @@ function getRemoteOrLocalUrl(manifest, key, isDetached) {
   // in detached apps, `manifest` is actually just app.json, so there are no remote url fields
   // we should return a local url starting with file:// instead
   if (isDetached) {
-    return _.get(manifest, key);
+    return get(manifest, key);
```

Comment:
```suggestion
    return manifest?.[key];
```

Reply:
Yeah, I though of this too, but then realized this file is JS 😮 🙀

Reply:
☠️